sanitiser/_id.js

```diff
@@ -18,23 +18,33 @@ function sanitize( raw, clean ){
   // error & warning messages
   var messages = { errors: [], warnings: [] };
 
-  // 'raw.id' can be an array
-  var rawIds = check.array( raw.id ) ? _.unique( raw.id ) : [];
-
-  // 'raw.id' can be a string
-  if( check.unemptyString( raw.id ) ){
-    rawIds.push( raw.id );
+  // 'raw.ids' can be an array if ids is specified multiple times
+  // see https://github.com/pelias/api/issues/272
+  var rawIdsString;
+  if (check.array( raw.ids )) {
+    rawIdsString = raw.ids[0];
+    messages.warnings.push( '`ids` parameter specified multiple times. Using first value.' );
```

Comment:
I'd rather see this as an error since the user might not see the warning, but it has a significant impact on results.